db/migrate/20240130163935_create_authors.rb

```diff
@@ -0,0 +1,10 @@
+class CreateAuthors < ActiveRecord::Migration[7.0]
+  def change
+    create_table :authors do |t|
+      t.string :name
+      t.string :email
```

Comment:
add unique index

Reply:
Added unique index to email attribute of authors table. Please check latest commit.